realm/realm-library/src/androidTest/java/io/realm/RealmIntegerTests.java

```diff
@@ -82,8 +83,38 @@ public void basic() {
      * if either arg is an int. That will cause sign extension.
      */
     @Test
-    public void getters() {
-        RealmInteger ri = new RealmInteger(0x5555444433332211L);
+    public void unmanaged_getters() {
```

Comment:
It is sorted into categories, by the function under test? In this case you get all `getters_<x>`, `setters_<x>` and so on instead of `unmanaged_<x>` and `managed_<x>`. You approach is isn't wrong as such and kinda makes sense since you have more heavy-weight unit tests than normal, but it just break the style of all other unit tests.